### ISSUE TITLE
Bug in Hfss.wave_port()

### ISSUE THREAD

**Issue by dcrawforAtAnsys** [ansys/pyaedt#3423]
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

`Hfss.wave_port()` fails when the first passed argument is an instance of `pyaedt.modeler.elements3d.FacePrimitive`.

In `hfss.py` on line 6247 

``` python
sheet_name = self.modeler.convert_to_selections(signal, True)[0]
``` 

returns the name of the 3D primitive to which `FacePrimitive` belongs. This ends up assigning the name of the 3d object to the variable `face` on line 6274.

### Steps To Reproduce

Pass a ``FacePrimitive`` that belongs to a 3d primitive to ``Hfss.wave_port()``

Will assign this to myself.

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.10

### Installed packages

Full PyAedt dependencies installed.

**Comment by dcrawforAtAnsys** [comment]
Automatic PEC creation gets the wrong type of argument when `face` is a 3D Object (line 6279 of `hfss.py`):

``` python
self._create_pec_cap(face, signal, -dist / 10)
```

**Comment by dcrawforAtAnsys** [comment]
Need to add support for the native API:
``` python
oModule.AutoIdentifyPorts(
	[
		"NAME:Faces", 
		269
	], True, 
	[
		"NAME:ReferenceConductors"
	], "1", True)
oModule.AutoCreatePECCapForWavePort(
	[
		"NAME:AutoCreatePECCapForWavePort",
		"Wave Port Name:="	, "1",
		"Face ID:="		, 269,
		"Thickness:="		, "0.5mm",
		"Flip Side:="		, False,
		"Cover Holes:="		, True
	])
```

**Comment by dcrawforAtAnsys** [comment]
Resolved in PR #3566
